pkg/config/config_template.yaml

```diff
@@ -340,6 +340,20 @@ api_key:
 #
 # use_proxy_for_cloud_metadata: false
 
+## @param auto_shutdown_method - string - optional - default: ""
+## Activate and select an automatic shutdown method: the Agent stops when some conditions are met.
+## Feature is only supported on POSIX systems.
+## Available method:
+## "noprocess": Detect when no other processes (non-agent) are running to trigger shutdown. `HOST_PROC` is taken into account when gathering processes.
+#
+# auto_shutdown_method: ""
+
+## @param auto_shutdown_validation_period - int - optional - default: 60
+## Time (in seconds) during which the auto shutdown will validate that the selected shutdown mechanism returns true.
```

Comment:
```suggestion
## Time (in seconds) during which the auto shutdown validates that the selected shutdown mechanism returns true.
```

Reply:
This phrasing is still not clear to me -- what does it mean to validate?  Why does it take so long?

If I understand the implementation, this is the time for which the shutdown condition must be met before the agent will shut down.

Reply:
We try to avoid future tense in the documentation when possible. If you think it changes the meaning, it can stay as is.

Reply:
I was commenting on the patch, not on your comment :)

Reply:
(in other words, I find it unclear both before and after the change in tense)

Reply:
It takes so long to improve reliability. For instance in the `noprocess` case, if the only other process restarts (for some reason) and the Agent runs the `check` at this moment, it could trigger an unwanted exit.